pyproject.toml

```diff
@@ -60,6 +60,12 @@ dependencies = [
     "dbt-adapters @ git+https://github.com/dbt-labs/dbt-adapters.git",
     "dbt-common @ git+https://github.com/dbt-labs/dbt-common.git",
 ]
+[tool.hatch.envs.default.scripts]
+docker-dev = [
```

Comment:
Linting and unit tests work with this, but it is not yet setup to connect to Postgres locally as the container can't see it. That's out of scope for now.

Reply:
Can we leave a comment / link to an issue ticket?